src/content/docs/nrql/using-nrql/query-time-range.mdx

```diff
@@ -61,7 +58,7 @@ You have a few ways to specify time in the `SINCE` and until `UNTIL` clauses.
 </Callout>
 
 ### Unix time (Epoch Milliseconds) [#time-n-units-ago]
-The time can be referenced as Unix time in milliseconds. This is the number of milliseconds since `1970-01-01T00:00:00Z`, or midnight on the 1st of January, 1970. For example:
+You can have the time reference as Unix time in milliseconds. This is the number of milliseconds since `1970-01-01T00:00:00Z`, or midnight on the 1st of January, 1970. For example:
```

Comment:
That first sentence is oddly structured. I needed the previous version to understand it.